ansible/roles/docker/templates/ubuntu1804_sharedlibs.Dockerfile.j2

```diff
@@ -58,7 +58,7 @@ ENV OPENSSL300DIR /opt/openssl-3.0.0
 
 RUN mkdir -p /tmp/openssl_3.0.0 && \
     cd /tmp/openssl_3.0.0 && \
-    curl -sL https://www.openssl.org/source/openssl-3.0.0-alpha14.tar.gz | tar zxv --strip=1 && \
+    git clone git@github.com:quictls/openssl.git && cd openssl && \
```

Comment:
```suggestion
    git clone https://github.com/quictls/openssl.git -b openssl-3.0.0-alpha14+quic --depth 1 && \
    cd openssl && \
```
Use alpha14 for now as alpha15 breaks some Node.js tests (https://github.com/nodejs/node/issues/38373).